doc/developer/doc/index.md

```diff
@@ -26,9 +26,9 @@ Deploying documentation
 -----------------------
 You can create the complete deployable documentation package by running
 
-    mvn clean site
+    ./gradlew clean javadoc
```

Comment:
I didn't even read the context of the rest of the documentation page when I updated that command.

Reading through it now, I see that even `mvn clean site` was incorrect.

It really should be `build-docs.py` but that needs a target directory as a parameter. I could make that optional and then a user could just run that command without any args to build the full documentation site.

Do you agree with that?

Reply:
Actually I think I will just make a `buildDocs` gradle task that calls the sub tasks and then executes jekyll.

Reply:
Fixed in bdcf304

Reply:
Thanks Eric. So the correct command is `./gradlew buildDocsSite` and it will build both the documentation and JavaDocs site? Could you update the `index.md` file here to reflect that?

Reply:
I tested out building the documentation with `./gradlew buildDocsSite` and copying `doc/_site` to a web server under `/irida-documentation`. It all works for me (jekyll documentation, JavaDocs, and Swagger API docs). So I think just updating the instructions in our documentation is needed.

Reply:
Fixed in 4bca8c61, I also changed the ci build to use this command instead of running them separately.